### ISSUE TITLE
Program difficulty (No restriction! Even logging "Hello, world!" is okay!)

### ISSUE THREAD

**Issue by hackleygithub** [wzhouwzhou/one-line-wonders#19]
Is there a requirement on the difficulty of the program or what the program has to accomplish?

**Comment by kannagikazuko** [comment]
No, any and all contributions are possible! The only requirement is for your program to accomplish something in one line (i.e. no using `;` instead of a new line). Open five Pull Requests and you can get a free hacktober shirt mailed for free while supplies last.

**Comment by wzhouwzhou** [comment]
I added a sample starting point for new contributors in the readme in #18

**Comment by kannagikazuko** [comment]
I have just merged #18, thank you @wzhouwzhou!

**Comment by wzhouwzhou** [comment]
I would also say imports/include lines where the language forces it (c++, python) can be exempt?

**Comment by kannagikazuko** [comment]
Agreed; we should add that to the README.md file.

**Comment by kannagikazuko** [comment]
I updated README.md in 4c436c83511dff045a96974fa8462c8969ffc4ae to reflect this.
